j5/backends/console/sr/v4/power_board.py

```diff
@@ -0,0 +1,146 @@
+"""Console Backend for the SR V4 power board."""
+
+from datetime import timedelta
+from typing import Dict, List, Optional, Type
+
+from j5.backends import Backend
+from j5.backends.console.env import Console, ConsoleEnvironment
+from j5.boards import Board
+from j5.boards.sr.v4.power_board import PowerBoard, PowerOutputPosition
+from j5.components import (
+    BatterySensorInterface,
+    ButtonInterface,
+    LEDInterface,
+    PiezoInterface,
+    PowerOutputInterface,
+)
+
+
+class SRV4PowerBoardConsoleBackend(
+    PowerOutputInterface,
+    PiezoInterface,
+    ButtonInterface,
+    BatterySensorInterface,
+    LEDInterface,
+    Backend,
+):
+    """The console implementation of the SR V4 power board."""
+
+    environment = ConsoleEnvironment
+    board = PowerBoard
+
+    @classmethod
+    def discover(cls) -> List[Board]:
+        """Discover boards that this backend can control."""
+        raise NotImplementedError("The Console Backend cannot discover boards.")
+
+    def __init__(self, serial: str, console_class: Type[Console] = Console) -> None:
+        self._serial = serial
+        self._output_states: Dict[int, bool] = {
+            output.value: False
+            for output in PowerOutputPosition
+        }
+        self._led_states: Dict[int, bool] = {
+            i: False
+            for i in range(2)
+        }
+
+        # Setup console helper
+        self._console = console_class(f"{self.board.__name__}({self._serial})")
+
+    @property
+    def firmware_version(self) -> Optional[str]:
+        """The firmware version reported by the board."""
+        return None  # Console, so no firmware
```

Comment:
Could we return the j5 version? Not sure if useful but might make debugging which version is in use easier?